README.md

```diff
@@ -1 +1,215 @@
-# capstone-ebike-reservation-frontend
\ No newline at end of file
+<div align="center">

```

Comment:
- [ ] Kindly, include the required `npm install` and `npm start` commands that your users will need to setup and run the frontend app. 👍

Reply:
✔️ DONE